go/runtime/registry/config.go

```diff
@@ -188,7 +189,7 @@ func newConfig(dataDir string, consensus consensus.Backend, ias ias.Endpoint) (*
 					Consensus:         consensus,
 					SandboxBinaryPath: sandboxBinary,
 					InsecureNoSandbox: insecureNoSandbox,
-				})
+				}, commonStore)
```

Comment:
I would rather put this into the `hostSgx.Config` structure, similar to the others.